### DISCUSSION TITLE
VDH failed to download video.

### DISCUSSION THREAD

**Discussion by jeh7354** [aclap-dev/video-downloadhelper#732] [Help and Q&A]
### Your question

VDH failed to download a specific media. I was using Brave (and earlier Firefox, where I have a license and coapp installed). No previous problems until this one. Any help is appreciated. Thanks. Copy of bug details follows:
{
  "version": "9.0.1.22",
  "target": "google",
  "channel": "stable",
  "page_url": "https://zonia.com/is-episode-1-live-4903467a",
  "error": {
    "details": "SideDownload error: : [dash @ 0000021de2a9b880] Error when loading first fragment of playlist\r\n[in#0 @ 0000021de2a9b6c0] Error opening input: Server returned 404 Not Found\r\nError opening input file https://video-node.swarmcdn.com/f78b4e69-52cb-4c35-bd18-700668a55ad8/52e7c3256d3f4f13ea09cd183b8153b80e388be0f0c70ba2b33cbcff77de08e5.mpd?t=0.3537006365316886.\r\nError opening input files: Server returned 404 Not Found\r\n",
    "downloadable_id": "downloadable_1120258292701356",
    "error": "coapp_failure",
    "id": "coapp_failure_f2ffe13f-22c9-4e03-ad64-2b1340894fbf"
  }
}

### Addon & CoApp version and details

Version 9.0.1.22 (stable), target google, locale en-US.

Companion App installed (v2.0.19) Recheck

C:\Program Files\DownloadHelper CoApp\vdhcoapp.exe

License not set

NOTE: the above details are from Brave browser. The media would not download in Firefox with license.

**Comment by Mattmjs** [comment]
I've looked at those videos before on the old google forum and indeed fails. But there is a solution.
@paulrouget , the solution is to change the url:

**original**
[https://video-node.swarmcdn.com/f78b4e69-52cb-4c35-bd18-700668a55ad8/52e7c3256d3f4f13ea09cd183b8153b80e388be0f0c70ba2b33cbcff77de08e5.**mpd**?t=0.3537006365316886]

**changed url** - mpd becomes m3u8
[https://video-node.swarmcdn.com/f78b4e69-52cb-4c35-bd18-700668a55ad8/52e7c3256d3f4f13ea09cd183b8153b80e388be0f0c70ba2b33cbcff77de08e5.**m3u8**?t=0.3537006365316886]

**Reply by jeh7354** [reply]
Thanks, Mattmjs
